tox.ini

```diff
@@ -66,5 +66,5 @@ commands =
     rm tests/__init__.py
     pycodestyle celery_utils tests
     pydocstyle celery_utils tests
-    isort --check-only --recursive tests test_utils celery_utils manage.py setup.py test_settings.py
```

Comment:
why did we need to remove `--recursive`

Reply:
Logs displayed an error saying --recursive is deprecated